docs/firefighting/index.md

```diff
@@ -0,0 +1,787 @@
+
+https://www.commcarehq.org/hq/admin/system/ - catchall system info, contains deploy history, pillowtop info, and a bunch of other stuff
+
+https://www.commcarehq.org/hq/admin/system/check_services - plaintext URL that checks the status of a bunch of support services
+
+# In case of a reboot
+
+## After reboot, whether or not it was deliberate
+
+In case a machine has automatically rebooted or you needed to reboot a machine,
+you will need to run the after-reboot protocol directly afterwards.
+You can specify a single server by IP address, a single server by its name in inventory.ini.
+You will have to confirm to run, and then provide the vault password for your environment.
+
+```bash
+$ cchq <env> after-reboot [all|<group>|<server>]
+Do you want to apply without running the check first? [y/N]y
+```
+
+You don't always control the reboot process (sometimes our provider will expectedly or unexpectedly reboot a VM), but if you do, here's the process end to end:
+
+```bash
+# Before shutting down VMs
+$ cchq <env> fab supervisorctl:'stop all'
+$ cchq <env> ansible-playbook stop_servers.yml
+
+# After restarting VMs
+$ cchq <env> after-reboot all
+$ cchq <env> fab manage:check_services  # ping various auxiliary services to make sure they're up
+  # if any services aren't running, you may have to manually start them:
+$ # cchq <env> run-module all service 'name=<service> state=started'
+$ cchq <env> fab restart_services  # start app processes
+```
+
+# In case of network outage
+
+If there has been a network outage in a cluster (e.g. firewall reboot), the following things should be checked to verify they are working:
+
+## Check services
+
+```bash
+$ ./manage.py check_services
+# or go to
+https://[commcarehq.environment.path]/hq/admin/system/check_services
+```
+
+## Check that change feeds are still processing
+
+You can use this graph on [datadog](https://app.datadoghq.com/dashboard/ewu-jyr-udt/change-feeds?to_ts=1549314000000&is_auto=false&live=true&from_ts=1549227600000&tile_size=m&page=0&fullscreen_widget=185100827)
+
+# Service Information
+
+Restarting services: `cchq <env> service <service_name> restart`
+
+Stopping services: `cchq <env> service <service_name> stop`
+
+Service logs: `cchq <env> service <service_name> logs`
+
+## Datadog Dashboards
+
+[postgres/pgbouncer](https://app.datadoghq.com/dash/263336/postgres---overview)
+
+[redis](https://app.datadoghq.com/dash/290868/redis-timeboard)
+
+[rabbitmq](https://app.datadoghq.com/screen/487480/rabbitmq---overview)
+
+[pillow](https://app.datadoghq.com/dash/256236/change-feeds-pillows)
+
+[celery/celerybeat](https://app.datadoghq.com/dash/141098/celery-overview)
+
+[elasticsearch](https://app.datadoghq.com/screen/127236/es-overview)
+
+[kafka](https://app.datadoghq.com/screen/516865/kafka---overview-cloned)
+
+[riak/riak-cs](https://app.datadoghq.com/dash/175518/riak-cs-system-resources)
```

Comment:
We should rename this to "Blob DB Dashboard"

I updated it so graphs relevant to all blob db backends are now at the top, and riak specific graphs are below. Also updated the name of the dashboard. The link can be changed to https://app.datadoghq.com/dashboard/753-35q-fwt/blob-db-dashboard (although the old link still works too).

I will do this in a separate PR once this is merged.

Reply:
https://github.com/dimagi/commcare-cloud/pull/2612/commits/537922107b3b71d1f31d18ebee44495be0ea3478